bin/oneoff/backfill_data/studio_people

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env ruby
+
+# This script creates a StudioPerson (if one does not already exist) for every teacher.
+
+require_relative '../../../dashboard/config/environment'
+
+# The user_id variable is used to track the in-progress user in case of an exception.
+user_id = 0
+
+begin
+  # Unfortunately, this query will not be peformant as the result of no index existing on
+  # users.user_type. As we are batching (via find_each), this should not impact live site traffic.
+  # And since we are willing to allow this script to run for hours, we are fine with this.
```

Comment:
Do you have a (general) estimate on how many hours this is expected to take?

Reply:
Eight?